grails-app/views/productAssociation/list.gsp

```diff
@@ -119,7 +119,7 @@
 
                                         <td>${fieldValue(bean: productAssociationInstance, field: "comments")}</td>
 
-                                        <td><format:date obj="${productAssociationInstance.dateCreated}" /></td>
+                                        <td><g:formatDate date="${productAssociationInstance.dateCreated}" /></td>
```

Comment:
Instead of refactoring format:date to account for the new java.time classes I opted to simply replace it with calls to g:formatDate. format:date should be deprecated anyways IMO since it doesn't have all the features that g:formatDate has and we should be trying to unify our approach to l10n.